arch/arm/src/stm32/stm32_comp.c

```diff
@@ -34,6 +46,8 @@
 
```

Comment:
The above comment is now incorrect.

I recommend:

```
/* This file is only a thin shell that includes the correct COMP
 * implementation. At this moment STM32 COMP IP versions 1 and 2 are
 * supported.
 *   - STM32 COMP IP version 1: SMT32F33XX
 *   - STM32 COMP IP version 2: SMT32G4XXX
 */
```

Reply:
It appears GitHub isn't showing the comment. It's on line 41 (was line 29) of arch/arm/src/stm32/stm32_comp.c and currently reads:

```
/* This file is only a thin shell that includes the correct COMP
 * implementation. At this moment only STM32 COMP IP version 1 device is
 * suportted.
 *   - STM32 COMP IP version 1: SMT32F33XX
 */
```

Reply:
Thanks for helping the review. Done